app/lib/shared/Core/entities/DocumentEntity/mandatoryTest_6_1_27_7.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @param {any} doc
+ */
+export default function (doc) {
+  /** @type {Array<{ message: string; instancePath: string }>} */
+  const errors = []
+  let isValid = true
+
+  const checkedDocumentCategories = new Set(['vex'])
+
+  if (!checkedDocumentCategories.has(doc.document?.category))
+    return { errors, isValid }
+
+  /** @type {unknown} */
+  const vulnerabilities = doc.vulnerabilities
+  if (Array.isArray(vulnerabilities)) {
+    vulnerabilities.forEach((vulnerability, vulnerabilityIndex) => {
+      if (!vulnerability.product_status) return
```

Comment:
As 6.1.27.6 (see previous review comment) does not longer covers the part of checking whether the product status is there, it must be implemented here.